crypto-intermediary/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "token-association-js-sdk",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "type": "module",
```

Comment:
If using `"module"` here, then please update to the `alice.js` and `bob.js` files `import` instead of `require`.

Or delete line 12 here to use `require` in the JS files.